objects/rct2tt/ride/rct2.tt.1920sand.json

```diff
@@ -32,22 +32,24 @@
             "sv-SE": "Art déco-matstånd",
             "ko-KR": "아르데코 음식 상점",
             "zh-TW": "裝置藝術食物攤位",
+            "ja-JP": "アールデコ食料品店",
             "pt-BR": "Quiosque de Art Déco de Comida",
             "cs-CZ": "Art Deco stánek",
             "pl-PL": "Kiosk art deco",
             "ru-RU": "Палатка Арт-Деко"
         },
         "description": {
-            "en-GB": "A stall selling Noodles and Fresh Lemonade",
+            "en-GB": "A stall selling noodles and fresh lemonade",
             "fr-FR": "Un restaurant proposant des pâtes et de la limonade",
-            "de-DE": "Ein Stand, der Nudeln und frische Limonade verkauft",
+            "de-DE": "Ein stand, der nudeln und frische limonade verkauft",
```

Comment:
This should be reverted; German uses uppercase to denote nouns.

Reply:
In general, please do not touch languages you don't speak very well.